operator/src/test/java/oracle/kubernetes/operator/http/rest/RestBackendImplTest.java

```diff
@@ -326,11 +328,48 @@ void whenPerClusterResourceReplicaSetting_scaleClusterUpdatesClusterResource() {
             .withReplicas(1);
     testSupport.defineResources(clusterResource);
 
+    configureDomain().withClusterReference(clusterResource.getMetadata().getName());
     restBackend.scaleCluster(DOMAIN1, CLUSTER_1, 5);
 
     assertThat(getUpdatedClusterResource().getSpec().getReplicas(), equalTo(5));
   }
 
+  @Test
+  void whenMultipleClusterResourceWithSameClusterName_scaleClusterUpdatesClusterInCorrectDomain() {
+    final ClusterResource clusterResource1 = createClusterResource(DOMAIN1, NS, CLUSTER_1)
```

Comment:
The two test cases have the similar setup logic, and can be refactored to use the same setup method with three domain names as the parameters.

Reply:
Thanks for the suggestion. Done.